### ISSUE TITLE
Investigate PurgeCSS for optimizing cSS

### ISSUE THREAD

**Issue by ccorda** [patronage/bubs-next#90]
https://purgecss.com/guides/next.html#customize-postcss-configuration-next-js-9-3
https://dev.to/thomasledoux1/shipping-minimal-css-with-next-js-bootstrap-3mdi

**Comment by kylehotchkiss** [comment]
My experience with larger next.config.js changes like this (like basic scss support before Next added it to core) from we first started with next.js is that they make updates to new versions very difficult. Next makes a lot of babel and webpack changes behind the scenes which tend to conflict. The per-project savings should be compared with difficulty of getting upstream next.js versions loaded which could offer various other compelling perf savings.

Getting existing projects on the latest next.js code would offer performance upgrades, and exploring dynamic imports for flex modules like we discussed in slack recently could also similar benefits

**Comment by ccorda** [comment]
I agree with you on ease of upgrades being a consideration, and that a custom postcss config is harder to upgrade.

On the other hand, I'd say this article made a good case for the value in CSS purge:
https://papyrus.dev/@PapyrusBlog/how-we-reduced-next.js-page-size-by-3.5x-and-achieved-a-98-lighthouse-score

I'd like to at least try it and see what the benefits are so we can compare.

**Comment by ccorda** [comment]
Looks like this is being closer to a part of Next:
https://twitter.com/hdjirdeh/status/1369709676271726599?lang=en
